packages/waku/src/router/create-pages.ts

```diff
@@ -350,6 +390,12 @@ export function createPages<
     },
     async (id, { unstable_setShouldSkip }) => {
       await configure();
+      if (id === 'root') {
+        if (rootItem?.render === 'dynamic') {
+          unstable_setShouldSkip();
+        } else unstable_setShouldSkip([]);
```

Comment:
```suggestion
        } else {
          unstable_setShouldSkip([]);
        }
```

Reply:
fixed... in a separate PR would you like me to turn on this rule: https://eslint.org/docs/latest/rules/curly

It would enforce this preference

Reply:
yeah, please try that.